website/docs/advanced/api_access.md

```diff
@@ -7,7 +7,7 @@ It is possible to integrate directly with the Admin API. In this guide we will e
 
 ## Step 1: Create API token {#step-1-create-api-token}
 
-Please refer to [_how to create API tokens_](../user_guide/api-token) on how to create an API token. You'll need a token with `Admin` level access for this to work.
+Please refer to [_how to create API tokens_](../user_guide/api-token) on how to create an API token. Alternatively, you can use a [personal access token](/reference/api-tokens-and-client-keys.mdx#personal-access-tokens), see [_creating a personal api token_](../user_guide/personal-api-token.mdx) to create your own tokens. You'll need either an `Admin` or `Personal API Token` for this to work.
```

Comment:
I wonder if this can be rewritten a little bit. Something like:

```suggestion
You'll need either an [admin token](../reference/api-tokens-and-client-keys.mdx#admin-tokens) or a [personal API token](../reference/api-tokens-and-client-keys.mdx#personal-access-tokens) for this to work. To create one, follow the steps in the [_how to create API tokens_](../user_guide/token.mdx) guide or the [_how to create personal access tokens_](../user_guide/personal-api-token.mdx) guide, respectively.
```

This also links directly to the file for the token guide, which is preferable.

Reply:
I like it!